_blueprints/build_concept.adoc

```diff
@@ -86,11 +86,11 @@ image::projectbuildconfigdataflow.png["Project Build Configuration Data Flow",fl
 
 The details of the deployment (Canary, Blue-Green or other) is determined by the build configuration and is out-of-scope of the ALM build system. Also, the build system contains no dependency management for build jobs on multi-repository projects. So there is no means of configuring followup builds in the ALM system. It is possible, though, that the build itself, configured in the build configuration files, triggers a followup build using some means, possibly a later webhook in the ALM system.
 
-*For private GitHub repositories*.footnote:[ see mailing list discussion here: https://www.redhat.com/archives/almighty-public/2016-October/msg00110.html&sa=D&ust=1477567580676000&usg=AFQjCNEA1AUUb8L54soCYBUBN5V75dq3Hw[https://www.redhat.com/archives/almighty-public/2016-October/msg00110.html&sa=D&ust=1477567580676000&usg=AFQjCNEA1AUUb8L54soCYBUBN5V75dq3Hw]], the project configuration contains a ssh secret key that is used to access the private repository. The user gets a notification with the public key and is noted that he has to add the public key to the deploy keys on his git provider. As this procedure is not specific for OpenShift or GitHub, the needed UI dialogs are also generic. As an option, if the repository and/or build provider supports this, the key exchange can be done automatically via a provided API.footnote:[ For example: https://developer.github.com/v3/repos/keys/#add-a-new-deploy-key[https://developer.github.com/v3/repos/keys/#add-a-new-deploy-key]].
+*For private GitHub repositories*.footnote:[ see mailing list discussion here: https://www.redhat.com/archives/fabric8-services-public/2016-October/msg00110.html&sa=D&ust=1477567580676000&usg=AFQjCNEA1AUUb8L54soCYBUBN5V75dq3Hw[https://www.redhat.com/archives/fabric8-services-public/2016-October/msg00110.html&sa=D&ust=1477567580676000&usg=AFQjCNEA1AUUb8L54soCYBUBN5V75dq3Hw]], the project configuration contains a ssh secret key that is used to access the private repository. The user gets a notification with the public key and is noted that he has to add the public key to the deploy keys on his git provider. As this procedure is not specific for OpenShift or GitHub, the needed UI dialogs are also generic. As an option, if the repository and/or build provider supports this, the key exchange can be done automatically via a provided API.footnote:[ For example: https://developer.github.com/v3/repos/keys/#add-a-new-deploy-key[https://developer.github.com/v3/repos/keys/#add-a-new-deploy-key]].
```

Comment:
that url seems broken and I don't recall what it was about ;/ better to remove it than try rename it to another 404 link